File: MODUL-5/Bahasa C/PRAK502-2310817120014-ALIYARAFFANAURAAYU.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){
//Lengkapi Function ini
    int Hasil = nilai1 - nilai2 ;

    return fabs(Hasil);
}
int mutlak(int angka){
//Lengkapi Function ini
if (angka<0) {
    angka = -angka;
    }
return angka ;
}

int main()
{
 int a,b,c,d;
 
 scanf("%d",&a);
 scanf("%d",&c);
 scanf("%d",&b);
 scanf("%d",&d);
 
 int Hasil = hitung(a,b) + hitung(c,d);
 printf("%d",mutlak(Hasil));
 return Hasil;
}

File: MODUL-5/Bahasa Python/PRAK502-2310817120014-ALIYARAFFANAURAAYU.py
def hitung(nilai1, nilai2):
    Hasil = nilai1 - nilai2
    return abs(Hasil)

def mutlak(angka):
    if angka < 0:
        angka = -angka
    return angka

a, c, b, d = input().split()
Hasil = hitung(int(a), int(b)) + hitung(int(c), int(d))
print(mutlak(Hasil))